src/scripts/components/widgets/FilesUploader.js

```diff
@@ -131,58 +123,58 @@ class FilesUploader extends Component<Props, Object> {
         file: file,
         fileName: file.name + ' | ' + file.size + 'bytes'
       })
+
+      this.props.history.push(FILESUPLOADER_PATH_TO)
     } else {
       this.props.checkUserWallet()
     }
   }
 
   onDrag (e: Object) {
-    // If wallet not secure open the modal
-    if (this.props.isWalletSecured) {
-      const status = e.type
-      let klass = ''
-      if (status === 'dragenter' || status === 'mouseover') {
-        klass = 'dragenter'
-      } else if (status === 'drop') {
-        klass = 'drop'
-      } else {
-        klass = ''
-      }
-      this.setState({
-        dragClass: klass
-      })
+    const status = e.type
+    let klass = ''
+    if (status === 'dragenter' || status === 'mouseover') {
+      klass = 'dragenter'
+    } else if (status === 'drop') {
+      klass = 'drop'
     } else {
-      this.props.checkUserWallet()
+      klass = ''
     }
+
+    this.setState({
+      dragClass: klass
+    })
   }
 
   renderUploadTrigger ({ card }: { card: boolean } = {}) {
     return (
       <Fragment>
-        <InputFile
-          data-test-id="upload-file-input"
-          onClick={this.onCheck}
-          onChange={this.onFileChosen}
-          onDragEnd={this.onDrag}
-        />
-        <UploadCover>
-          {card && <Title>Upload video</Title>}
-          <UploadCoverIcon>
-            {card && (
-              <Icon>
-                <FilesUploaderSvg />
```

Comment:
removing this the card layout is now missing the svg icon
<img width="442" alt="schermata 2018-06-06 alle 09 30 51" src="https://user-images.githubusercontent.com/362016/41023754-8f900f06-696d-11e8-8cae-9f77a6257af7.png">

Reply:
My bad, @eliawk 
Fixing that